### PR TITLE
(Test Demo 3) Separate component, connector and resolver tests

### PR DESCRIPTION
**Pull request by robbiecarlton**

### CODE REVIEW

File: src/graphql-server/resolvers.test.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import resolvers from './resolvers'
```
Comment (danekszy) [inline src/graphql-server/resolvers.test.js]: It calls the actual zomes, right?

File: src/components/HappHosting/HappHosting.connector.test.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+            hash: 'hash2'
+          }
+        ]
+      }
+    }
+  }
+]
+
+describe('connector', () => {
+  it('runs the AllHapsQuery', async () => {
```
Comment (danekszy) [inline src/components/HappHosting/HappHosting.connector.test.js:37]: Misleading test name - we're checking the results and not if the query actually executed :P

Comment (robbiecarlton) [reply src/components/HappHosting/HappHosting.connector.test.js:37]: hmm. debatable. Actually all we care about here is that the query ran. So what's misleading is all those `expects`, we probably just want one.

Comment (danekszy) [reply src/components/HappHosting/HappHosting.connector.test.js:37]: Then I would *expect* :P `toHaveBeenCalled` in expect ;)

**Review by lorenjohnson — APPROVED** [review]
I prefer this one of the 3 👍

**Pull request closed without merging**